pkg/nimble/netif/include/nimble_netif.h

```diff
@@ -102,6 +102,26 @@ extern "C" {
 #define NIMBLE_NETIF_MTU            (1280U)
 #endif
 
+/**
+ * @brief   Set to 1 to enforce connection interval spacing
+ *
+ * Enabling this option will enforce that every BLE connection a node maintains,
+ * independent of the nodes role, uses a different connection interval, with a
+ * minimum spacing between connection intervals as defined by
+ * NIMBLE_NETIF_CONN_ITVL_SPACING_MS.
+ */
+#ifndef NIMBLE_NETIF_FORCE_CONN_ITVL_SPACING
+#define NIMBLE_NETIF_FORCE_CONN_ITVL_SPACING    1
+#endif
+
+/**
+ * @brief   Minimum spacing of connection interval when using randomized
+ *          intervals, in multiples of 1.25ms
+ */
+#ifndef NIMBLE_NETIF_CONN_ITVL_SPACING_MS
+#define NIMBLE_NETIF_CONN_ITVL_SPACING_MS       2
```

Comment:
True, the `_MS` suffix does not really apply here. I just saw that it would further simplify things if we simply merge `NIMBLE_NETIF_CONN_ITVL_SPACING_MS` and `NIMBLE_NETIF_FORCE_CONN_ITVL_SPACING` into a single variable. Will fix.

Reply:
Also, the internal code of the NimBLE controller will convert this value to microseconds, hence a value o 2 here will lead to 2500 in the controller, circumventing the floating issue :-)

This 1.25ms factor is something that the Bluetooth standard defines, and this is also used in the HCI interface commands. So this int value is plainly propagated in the BLE Host.